CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ IMPROVEMENTS:
  * agent: Removed leading whitespace from JSON-formatted log output. [[GH-9795](https://github.com/hashicorp/nomad/issues/9795)]
  * cli: Added optional `-task <task-name>` flag to `alloc logs` to match `alloc exec` [[GH-10026](https://github.com/hashicorp/nomad/issues/10026)]
  * cli: Improved `scaling policy` commands with -verbose, auto-completion, and prefix-matching [[GH-9964](https://github.com/hashicorp/nomad/issues/9964)]
+ * cli: Fixed a bug where `nomad operator debug` incorrectly parsed https Consul API URLs. [[GH-9625](https://github.com/hashicorp/nomad/pull/10082)]
```

Comment:
The 1.0.4 release build has already been cut; this will need to go into 1.1 unless it's a regression from 1.0.3.

Reply:
I believe this has been broken from the beginning, so I understand the decision.  I rebased and moved the change up to the 1.1.0 (Unreleased) section.